Add AR0820 and AR0830 sensor configurations for IPU6

ijulipan opened the pull request:
Add support for AR0820 and AR0830 sensors:
- Add AR0820-1 sensor configuration (GMSL, 3840x2160 UYVY)
- Add AR0830-1 sensor configuration (3840x2160 UYVY)
- Add AR0830-2 sensor configuration (3840x2160 UYVY)
- Register new sensors in libcamhal_profile.xml

These configurations enable camera HAL support for AR0820 and AR0830 sensors on Intel IPU6 EPMTL platform with proper media controller pipeline setup and V4L2 format definitions.

hao-yao commented on config/linux/ipu6epmtl/libcamhal_profile.xml:
```diff
@@ -30,6 +30,7 @@
                                  lt6911uxc,lt6911uxe-1-0,lt6911uxe-2-4,
-                                 external_source,ar0234_usb"/>
+                                 external_source,ar0234_usb,
+                                 ar0820-1-0, ar0830-1-0, ar0830-2-4"/>
```
Please move this before external_source.

ijulipan replied:
done.

ijulipan committed:
ipu6-camera-hal: Add AR0820 and AR0830 sensor configurations for IPU6 EPMTL

Add support for AR0820 and AR0830 sensors:
- Add AR0820-1 sensor configuration (GMSL, 3840x2160 UYVY)
- Add AR0830-1 sensor configuration (3840x2160 UYVY)
- Add AR0830-2 sensor configuration (3840x2160 UYVY)
- Register new sensors in libcamhal_profile.xml

These configurations enable camera HAL support for AR0820 and AR0830
sensors on Intel IPU6 EPMTL platform with proper media controller
pipeline setup and V4L2 format definitions.

Signed-off-by: Izzul Irfan <muhammadx.izzul.irfan.bin.che.iswanizam@intel.com>
